app/src/main/java/com/nervousfish/nervousfish/ConstantKeywords.java

```diff
@@ -15,6 +15,8 @@
     public static final String WAIT_MESSAGE = "Wait message";
     public static final String DATA_RECEIVED = "Data received";
     public static final String TAP_DATA = "Tap data";
+    public static final String CHOOSE_VERIFICATION_PREFERENCE = "choose_verification_method_every_time";
+    public static final String DISPLAY_NAME = "display_name";
```

Comment:
Because the user will never see that string, it's only used internally because some Android libraries don't support enums for some reason and require you to use Strings to identify things.